scripts/load.py

```diff
@@ -75,17 +78,21 @@
     # here, we combine the subnet and the random bits
     # to get an IP address from the previously specified subnet
     addr = IPv4Address(subnet.network_address + bits)
+    addr6 = random.choice(ipv6)
     addr_str = str(addr)
+    addrs = [addr_str, addr6]
     # have to add a space at the end for log formatting
     remoteAddr = addr_str + " "
+    httpXForwardedForS = addr6
 
     # --Logfile Generation--
     twohundreds = 100 - (fourErrs + fiveErrs)
     statusList = [200] * twohundreds + [404] * fourErrs + [503] * fiveErrs
     status = random.choice(statusList)
-    log = host + remoteAddr + "- " + remoteUser + timeLocal + request + str(status) + " " + bodyBytesSent + httpReferer + \
-        httpUserAgent + httpXForwardedFor + requestTime + \
-        upstreamResponseTime + upstreamConnectTime
+    log = remoteAddr + str(status) + " " + httpXForwardedForS
```

Comment:
Why change the log format in the testing script? Also, why leave commented code if it is changed?

Reply:
I can clean that up, and I changed the log format in the testing script because that's the log format we're using internally.  I can revert that too if you want.